storage/range_tree.go

```diff
@@ -73,8 +63,25 @@ func SetupRangeTree(batch engine.Engine, ms *engine.MVCCStats, timestamp proto.T
 	return nil
 }
 
-// GetRangeTree fetches the RangeTree proto.
-func getRangeTree(db *client.KV) (*rangeTree, error) {
+// flush writes all updated nodes and the tree to the transaction.
+func (t *treeContext) flush() error {
+	if t.treeUpdated {
```

Comment:
Insert can only be called as part of the AdminSplit transaction.  Do I need any other special treatment here?  (specifically the clearing of the cache?)